src/main/java/com/google/idea/perf/Matching.kt

```diff
@@ -86,8 +134,8 @@ fun fuzzyMatch(source: String, pattern: String): MatchResult {
     var column = 0
     var maxScore = Int.MIN_VALUE
 
-    for (r in pattern.indices) {
-        for (c in source.indices) {
+    for (r in 0..pattern.length) {
```

Comment:
Was this was accidentally changed when rebasing?

Reply:
This is intended because `pattern.length` must be included in the range

Reply:
Whoops, you're right! I did not realize that `..` denoted an inclusive range!